deepinv/physics/__init__.py

```diff
@@ -34,6 +34,7 @@
     SaltPepperNoise,
     ZeroNoise,
     FisherTippettNoise,
+    RicianNoise,
```

Comment:
Thanks for contributing RicianNoise!

Please can you also add the noise to the User guide docs, in the [noise table](https://github.com/deepinv/deepinv/blob/main/docs/source/user_guide/physics/physics.rst?plain=1#L153)

And please can you also add a test to the [noise tests file](https://github.com/deepinv/deepinv/blob/main/deepinv/tests/test_noise_model.py)? They test simple things like the random generator working correctly, the noise can be composed correctly etc.

Thanks!